Documentation/instructions.txt

```diff
@@ -57,7 +57,8 @@ Instructions:
     'php artisan migrate'
 
 12. You can seed the database width
-    'php artisan db:seed'
+    'php artisan db:seed' // This shouldn't work for now.
```

Comment:
This is obviously a WIP comment.  Making a comment here just so we make sure to check for these when it is final review time.